src/linter/linter.zig

```diff
@@ -125,6 +126,31 @@ pub const Linter = struct {
                 };
             }
         }
+
+        // Check each line
+        for (rules) |rule_with_severity| {
```

Comment:
We shouldn't have rules that iterate line-by-line. I'll add a `runOnce` method to `Rule`'s vtable

Reply:
Added in #241